traveler-swift-core/TravelerKit/Models/BookingItem.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-public struct BookingItem: CatalogItem, Decodable, Product {
```

Comment:
@jack-mo Please ensure these changes are captured in your BookingItem refactorings